rust/datafusion/src/logical_plan/dfschema.rs

```diff
@@ -136,12 +136,11 @@ impl DFSchema {
 
     /// Find the index of the column with the given name
     pub fn index_of(&self, name: &str) -> Result<usize> {
-        for i in 0..self.fields.len() {
-            if self.fields[i].name() == name {
-                return Ok(i);
-            }
+        if let Some(i) = self.fields.iter().position(|f| f.name() == name) {
+            Ok(i)
```

Comment:
small style thing but I guess we could do it roughly like this instead? 

```rust
    self.fields.iter().position(|f| f.name() == name)
        .ok_or_else(|| DataFusionError::Plan(format!("No field named '{}'", name)))
```

Reply:
Thanks. Fixed.